src/compile/data/bin.ts

```diff
@@ -38,15 +38,18 @@ function getSignalsFromModel(model: Model, key: string) {
   };
 }
 
+// Bad Name?
```

Comment:
What would be a better name?

Reply:
My bad. I didn't pay attention to the type of the argument.